must-gather/Dockerfile

```diff
@@ -5,5 +5,6 @@ COPY collection-scripts /usr/bin/
 
 RUN mkdir -p  /templates
 COPY templates /templates
+RUN ["chmod", "+x", "/usr/bin/gather_common_ceph_resources"]
```

Comment:
why are you doing this.

Reply:
Looks like the `+x` is missing in the file script he commited. Just run this locally and `git add` the change.

Reply:
yes, thanks, I was not aware of how to deal perm issue while creating new file, I was confused because the files are copied into a docker file, it was an easy fix, thanks